components/user/themes/UserTabs.js

```diff
@@ -46,16 +46,49 @@ export default function UserTabs({ data, BASE_URL }) {
   });
   const [tabs, setTabs] = useState(displayTabs);
   const changeTab = (e, value) => {
-    e.preventDefault();
+    e.preventDefault();    
     setTabs(
-      tabs.map((tab) =>
-        tab.name === e.target?.value || tab.name === value
-          ? { ...tab, current: true }
-          : { ...tab, current: false }
+      tabs.map((tab) =>{
+        if (tab.name === e.target?.value || tab.name === value){
+        addTabsToURL(tab.url)
+          return { ...tab, current: true }
+        } else {
+          return { ...tab, current: false }}
+        }
       )
```

Comment:
```suggestion
      }),
```

Reply:
i don't understand this change, looks like it has nothing to do with my PR

Reply:
Please use prettier